modules/log-collector-http-server.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log-collector-http-server_{context}"]
+= Configuring the collector to listen for connections as an HTTP server
+
+You can configure your log collector to listen for HTTP connections and receive audit logs as an HTTP server by specifying `http` as a receiver input in the `ClusterLogForwarder` custom resource (CR). This enables you to use a single common log store for audit logs that are collected from both inside and outside of your {product-title} cluster.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {oc-first}.
+* You have installed the {clo}.
+* You have created a `ClusterLogForwarder` CR.
+
+.Procedure
+
+. Modify the `ClusterLogForwarder` CR to add configuration for the `http` receiver input:
++
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1beta1
+kind: ClusterLogForwarder
+metadata:
+# ...
+spec:
+  serviceAccountName: <service_account_name>
+  inputs:
+    - name: http-receiver # <1>
+      receiver:
+        type: http # <2>
+        http:
+          format: kubeAPIAudit # <3>
+          port: 8443 # <4>
+  pipelines: # <5>
+    - name: http-pipeline
+      inputRefs:
+        - http-receiver
+# ...
+----
+<1> Specify a name for your input receiver.
+<2> Specify the input receiver type as `http`.
+<3> Currently, only receiving audit logs in the `kube-apiserver` webhook format is supported.
```

Comment:
When I first read this, I was expecting it to only support receiving (as opposed to sending or something). I think that "only" needs to move closer to what the limitation is. Maybe:

```suggestion
<3> Currently, it is supported to receive audit logs only in the `kube-apiserver` webhook format.
```

or maybe

```suggestion
<3> Currently, only the the `kube-apiserver` webhook format is supported.
```